xarray/tests/test_dask.py

```diff
@@ -1129,7 +1129,8 @@ def test_unify_chunks(map_ds):
 def test_unify_chunks_shallow_copy(obj, transform):
     obj = transform(obj)
     unified = obj.unify_chunks()
-    assert_identical(obj, unified) and obj is not obj.unify_chunks()
+    assert_identical(obj, unified)
+    # assert obj is not obj.unify_chunks()
```

Comment:
```suggestion
    assert obj is not obj.unify_chunks()
```

I think this may be right